requirements/requirements.txt

```diff
@@ -0,0 +1,8 @@
+ovos-lingua-franca>=0.4.4a2
+neon-stt-plugin-polyglot
```

Comment:
The Plugins and LF dependency should be removed here; they are implemented in other modules and not in the skill directly